CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ add_definitions(
     -std=c++11
     -Wall
     -pedantic
+    -Wno-old-style-cast
```

Comment:
```
cl : Command line error D8021: invalid numeric argument '/Wno-old-style-cast' [C:\Users\travis\build\euskadi31\statsd-cpp\build\statsd.vcxproj]
```

Reply:
Hi, I have revert the commit that make build failed